Gemfile

```diff
@@ -2,53 +2,53 @@ ruby File.read(".ruby-version").strip
 
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0"
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
 
-gem "aws-sdk-s3", "~> 1"
-gem "bootsnap", "~> 1"
-gem "gds-api-adapters", "~> 67"
-gem "gds-sso", "~> 15"
-gem "govspeak", "~> 6.5"
-gem "govuk_app_config", "~> 2"
-gem "govuk_publishing_components", "~> 21"
-gem "govuk_sidekiq", "~> 3"
-gem "hashdiff", "~> 1.0"
-gem "image_processing", "~> 1"
-gem "interactor", "~> 3"
-gem "kaminari", "~> 1"
-gem "mail-notify", "~> 1"
-gem "pdf-reader", "~> 2"
-gem "pg", "~> 1"
-gem "plek", "~> 4"
-gem "rinku", "~> 2"
-gem "rubyzip", "~> 2", require: "zip"
-gem "sanitize", "~> 5"
-gem "sass-rails", "< 6"
-gem "sidekiq-scheduler", "~> 3"
-gem "uglifier", "~> 4"
-gem "with_advisory_lock", "~> 4"
+gem "aws-sdk-s3"
```

Comment:
It'd be useful to have the commit message that is removing the pins to explain why they're being removed.